packages/adapter-test/index.ts

```diff
@@ -287,7 +300,7 @@ export function runBasicTests(options: TestOptions) {
     expect(dbAccount).toBeNull()
   })
 
-  test("deleteUser", async () => {
+  testIf(!skipTests?.includes("deleteUser"))("deleteUser", async () => {
```

Comment:
Why are you adding all this logic to skip the `deleteUser` test? It does seem to be implemented here, right?

Reply:
@ndom91 I'm a bit confused. The history shows that you added the logic you're asking to be removed. Can you explain?